Resources/views/CRUD/edit_orm_one_to_many_inline_table.html.twig

```diff
@@ -23,7 +23,7 @@ file that was distributed with this source code.
                             style="display:none;"
                         {% endif %}
                     >
-                        {{ nested_field.vars.label|trans({}, nested_field.vars.translation_domain) }}
+                        {{ nested_field.vars.label|trans({}, nested_field.vars['sonata_admin'].admin.translationDomain|default(nested_field.vars.translation_domain)) }}
```

Comment:
No idea how to solve this in 80 chars per line without destroying readability :/